frontend/src/scenes/trends/personsModalLogic.tsx

```diff
@@ -262,6 +272,20 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
 
                 return peopleResult
             },
+            loadPeopleFromUrl: async ({ url, stepNumber, breakdown, breakdown_value, label }) => {
```

Comment:
Do you think we could parse some of this directly from the URL itself? That would reduce this function to just requiring a `url`

Reply:
Yeah was thinking about that. These additional args are specific things for generating the label for display. I wouldn't want to parse the url, if smells. I don't want to start coupling things like this to what can be, at the moment, be treated as an opaque url.

If we really care about making sure that the label consistent with the data returned, we can either pass along an object along with the url, containing the info (could be the exact label although that seems a little prescriptive for what feels contextual to the consumer). Or we can return this data within the response?